pkg/terraform/action.go

```diff
@@ -105,9 +105,6 @@ type Instruction struct {
 
 // TerraformFields represent fields specific to Terraform
 type TerraformFields struct {
-	// AutoApprove will be deprecated in a later release, it is no longer used, --auto-approve=true is always passed to terraform
```

Comment:
I'm having a bit of trouble following the combination of flags that terraform wants now for an automated run.

https://www.terraform.io/docs/cli/commands/apply.html

Let's make sure to test this out with a real tf run through a bundle to make sure we are specifying (or not) all the right flags so that it doesn't prompt.

Reply:
Note that the `autoApprove` flag will still be inserted on applicable actions, e.g. https://github.com/vdice/porter-terraform/blob/6f714ec43034e0f2c05efdfa72cc0082cab82427/pkg/terraform/install.go#L30.  This PR just removes the ability for users to define it in a step.